src/components/KeyboardAvoidingView/index.ios.js

```diff
@@ -1,13 +1,74 @@
 /*
  * The KeyboardAvoidingView is only used on ios
  */
-import React from 'react';
-import {KeyboardAvoidingView as KeyboardAvoidingViewComponent} from 'react-native';
+import React, {useMemo} from 'react';
```

Comment:
If they plan to build it following the same API and behavior React Native has - it will be identical.

Reply:
It's really nice to see how `useAnimatedKeyboard` can be used in cases just like this one. Since the hook itself resolves one of major pain points of the React Native, it makes sense to go one step forward and include the `KeyboardAvoidingView` component in the library as well, so that the whole community can benefit from it, instead of just embedding it into the source code of the E/App.

@terrysahaidak Would you be open to submitting this part of the code as a PR to react-native-reanimated? We'll be more than happy to review and test it or even help with the implementation. Ideally, it should expose the same API as in the one from React Native so it can be used as a drop-in replacement, but if that's a limitation, we can think of a better API, just let me know if you have any ideas.